test/integration/eventBridge.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prefer-arrow-callback */
```

Comment:
Let's not disable lint rules. They're to ensure consistency among code-base and pick errors which could be picked by static analysis.

If however some lint rule makes something not possible to achieve, please elaborate, disabling in such rare scenarios of course makes sense

Reply:
Took me a while to remember why this is here, this might actually be a result of a conflict with my local `eslint` settings as it is resolving/ignoring a violation that was already present. I'll take a look and justify if required.